### PR TITLE
Upgrade packages to reach Composer 2 compatibility

### PR DESCRIPTION
**Pull request by greg0ire**
Upgraded packages:
- `doctrine/coding-standard`
- `ocramius/package-versions`

New package required: `composer/package-versions-deprecated`

Also: I clicked the button to migrate from travis-ci.org to travis-ci.com.

And finally, I prevent the installation of `doctrine/migrations` 2.3.0, this needs to be fixed in a separate PR

### CODE REVIEW

File: src/DoctrineORMModule/Collector/SQLLoggerCollector.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 use Laminas\DeveloperTools\Collector\AutoHideInterface;
 use Laminas\DeveloperTools\Collector\CollectorInterface;
 use Laminas\Mvc\MvcEvent;
+
 use function count;
```
Comment (TomHAnderson) [inline src/DoctrineORMModule/Collector/SQLLoggerCollector.php:12]: use function empty;?

File: test/DoctrineORMModuleTest/Paginator/AdapterTestIgnore.php [hunk 2]
```diff
@@ -2,86 +2,81 @@
...
 
         $this->createDb();
         $loader = new FixtureLoader();
         $loader->addFixture(new TestFixture());
-        $purger = new ORMPurger();
+        $purger   = new ORMPurger();
         $executor = new ORMExecutor($this->getEntityManager(), $purger);
         $executor->execute($loader->getFixtures());
 
         $this->qb = $this
```
Comment (TomHAnderson) [inline test/DoctrineORMModuleTest/Paginator/AdapterTestIgnore.php]: spell out $queryBuilder

**Review by TomHAnderson — APPROVED** [review]
Two minor changes; I'll push these to the PR

File: src/DoctrineORMModule/Collector/SQLLoggerCollector.php (hunk 1 above)
Comment (TomHAnderson) [reply src/DoctrineORMModule/Collector/SQLLoggerCollector.php:12]: This is not an issue.  Rather it throws php errors

**Review by SenseException — APPROVED** [review]

File: test/DoctrineORMModuleTest/Paginator/AdapterTestIgnore.php (hunk 2 above)
Comment (greg0ire) [reply test/DoctrineORMModuleTest/Paginator/AdapterTestIgnore.php]: Done :)

**Pull request merged**